Firebase.Storage/Packages/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "com.rest.firebase.authentication": "file:E:/Dev/Firebase/com.rest.firebase.authentication/Firebase.Authentication/Packages/com.rest.firebase.authentication",
+    "com.rest.firebase.authentication": "https://github.com/RageAgainstThePixel/com.rest.firebase.authentication.git#upm",
```

Comment:
Just revert or remove this. you can make it relative if needed, but if removed should pull dep from package.json anywayl

Reply:
Done